src/components/stegvisning/stegvisning-hooks.tsx

```diff
@@ -24,8 +34,61 @@ export const useStegvisningNavigation = (currentPath: Path) => {
         : stegvisningOrder
     navigate(stepArrays[stepArrays.indexOf(currentPath) + 1])
   }
+
   const onStegvisningPrevious = () => {
-    navigate(-1)
+    let antallStepTilbake = 1
+    const isEndring = loependeVedtak && isLoependeVedtakEndring(loependeVedtak)
+    const stepArrays = isEndring ? stegvisningOrderEndring : stegvisningOrder
+
+    const currentPathIndex = stepArrays.indexOf(currentPath)
+
+    // Hvis brukeren er på eller forbi afp steget
+    if (
+      (!isEndring && currentPathIndex >= 4) ||
+      (isEndring && currentPathIndex >= 2)
+    ) {
```

Comment:
Å hardkode indeksene føles som kan brekke lett. Hadde det vært bedre å slå opp i `stepArrays`? (Flere tilfeller)
Blir vel noe sånt som dette tror jeg:
```suggestion
    if (currentPathIndex >= stepArrays.indexOf(paths.afp)) {
```

Reply:
Det blir egentlig det samme. Denne logikken er tett koblet til innhold og rekkefølgen i `stegvisningOrderEndring` og `stegvisningOrder` og vil bli utdatert om noe redigeres der. Men jeg kan endre det for lesbarhetens skyld.